emath/src/vec2.rs

```diff
@@ -80,7 +82,18 @@ impl From<&Vec2> for (f32, f32) {
         (v.x, v.y)
     }
 }
-
+// ----------------------------------------------------------------------------
+// Compatibility and convenience conversions to and from Pos2:
+impl From<Pos2> for Vec2 {
+    fn from(pos: Pos2) -> Self {
+        vec2(pos.x, pos.y)
+    }
+}
+impl From<Vec2> for Pos2 {
+    fn from(pos: Vec2) -> Self {
+        pos2(pos.x, pos.y)
+    }
```

Comment:
I'm not sure this is a good idea. There are many places where we take `impl Into<Vec>` as an argument for e.g. sizes, and passing in a position there is probably a mistake, and should not compile. I think it is better if we keep conversations between positions and vectors explicit

Reply:
sure thing